datanommer.commands/datanommer/commands/__init__.py

```diff
@@ -331,3 +331,23 @@ def formatter(key, val):
         results.append(formatter(result.category, result))
 
     click.echo("[%s]" % ",".join(results))
+
+
+# Set the version
+try:
+    import importlib.metadata  # pragma: no cover
```

Comment:
You don't need to add `pragma: no cover` to every line, you can add it to branch points instead and then everything in that branch won't be counted.